docs/framework/guides/tutorials/implementing-an-inline-widget.md

```diff
@@ -694,11 +694,11 @@ export default class PlaceholderEditing extends Plugin {
 	}
 
 	_defineConverters() {
-		// ...
+		// Previously defined converters.
 	}
 
 	_defineSchema() {
-		// ...
```

Comment:
We should keep those ellipses below the comment, as shown here: https://github.com/cksource/ckeditor5-internal/issues/2307.